Cargo.lock

```diff
@@ -1523,8 +1527,8 @@ dependencies = [
  "httparse",
  "httpdate",
  "itoa",
- "pin-project 1.0.5",
- "socket2",
+ "pin-project",
+ "socket2 0.4.0",
```

Comment:
Two versions of `socket2`?

Reply:
We can't do anything about this. `hyper` needs the new one, while `trust-dns-resolver` uses the old one.